source/frontend/src/features/mapSideBar/acquisition/AcquisitionContainer.tsx

```diff
@@ -132,6 +136,9 @@ export const AcquisitionContainer: React.FunctionComponent<IAcquisitionContainer
   const fetchAcquisitionFile = useCallback(async () => {
     const retrieved = await retrieveAcquisitionFile(acquisitionFileId);
     if (exists(retrieved)) {
+      if (isValidId(retrieved.projectId)) {
+        retrieved.project = await getProjectFunction(retrieved.projectId);
+      }
```

Comment:
Not to do this now - but this could would benefit from a `Promise.all(...)` approach to fetch the related entities. Right now we wait for each request to complete in sequence. Putting a TODO comment for now would be fine with me